examples/base-vite-react-tailwind-ts/src/components/Button.tsx

```diff
@@ -1,4 +1,4 @@
-import * as React from 'react';
+import React from 'react';
```

Comment:
This was correct 

```suggestion
import * as React from 'react';
```

x the others